grun.sh

```
@@ -0,0 +1,2 @@
+#/bin/sh
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)